acme_diags/acme_diags_driver.py

```diff
@@ -1,24 +1,13 @@
 #!/usr/bin/env python
 from __future__ import print_function
-from typing import Dict, Tuple
-import os
-
-# Must be done before any CDAT library is called.
-os.environ["UVCDAT_ANONYMOUS_LOG"] = "no"
-os.environ["CDAT_ANONYMOUS_LOG"] = "no"
-# Needed for when using hdf5 >= 1.10.0,
-# without this, errors are thrown on Edison compute nodes.
-os.environ["HDF5_USE_FILE_LOCKING"] = "FALSE"
-# Used by numpy, causes too many threads to spawn otherwise.
-os.environ["OPENBLAS_NUM_THREADS"] = "1"
-os.environ["OMP_NUM_THREADS"] = "1"
```

Comment:
These env vars are now in `acme_diags/__init__.py`

Reply:
@chengzhuzhang It looks like we can set these env vars globally before any of their related imports.

Let me know if they should only be set through `acme_diags_driver`.